tests/test_build_spec.py

```diff
@@ -1,12 +1,12 @@
 import os
 from pathlib import Path
+
 from bioimageio.spec import maybe_convert_model, schema
-from bioimageio.spec.utils import yaml
+from bioimageio.spec.common_utils import yaml
+from bioimageio.spec.latest.build_spec import _get_local_path, build_spec
```

Comment:
This is just a minor point, but it's good practice to import the functionality being tested in the test instead of importing it globally, so that potential ImportErrors are caught on an individual test level.